src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -558,14 +578,38 @@ void IgnRenderer::Initialize()
   // setting the size and calling PreRender should cause the render texture to
   // be rebuilt
   this->dataPtr->camera->PreRender();
-  this->textureId = this->dataPtr->camera->RenderTextureGLId();
+
+  // Update the render interface (texture)
+  this->dataPtr->rhi->Update(this->dataPtr->camera);
 
   // Ray Query
   this->dataPtr->rayQuery = this->dataPtr->camera->Scene()->CreateRayQuery();
 
   this->initialized = true;
 }
 
+/////////////////////////////////////////////////
+void IgnRenderer::SetGraphicsAPI(const rendering::GraphicsAPI &_graphicsAPI)
+{
+  // Create render interface and reset params
+  this->dataPtr->rhiParams.clear();
+
+  if (_graphicsAPI == rendering::GraphicsAPI::OPENGL)
+  {
+    qDebug().nospace() << "Creating ign-renderering interface for OpenGL";
```

Comment:
`renderering` -> `rendering`

Reply:
Updated in ca52977561c63bb615e01b25fa62b350645dfda6